### ISSUE TITLE
Véhicule/Vélo électrique

### ISSUE THREAD

**Issue by fjg** [betagouv/ecolab-transport#13]
La base carbone distingue : 

- véhicule thermique moyen (0,253 CO2e kg/km)

- véhicule électrique entrée de gamme (0,0953 CO2e kg/km)
- véhicule électrique coeur de gamme (0,103 CO2e kg/km)
- véhicule électrique haut de gamme (0,139 CO2e kg/km)
- véhicule hybride Mild entrée de gamme (0,171 CO2e kg/km)
- véhicule hybride Mild coeur de gamme (0,217-0,232 CO2e kg/km) - (diesel-essence)
- véhicule hybride Mild haut de gamme (0,277-0,348 CO2e kg/km) - (diesel-essence)
- véhicule hybride Full hybrid Prius coeur de gamme (0,147 CO2e kg/km)
- véhicule hybride Full hybrid Prius haut de gamme (0,207 CO2e kg/km)
- véhicule hybride Full hybrid P2 coeur de gamme (0,183 CO2e kg/km)
- véhicule hybride Full hybrid P2 haut de gamme (0,255 CO2e kg/km)

Ces distinctions nuisent à la lecture et compréhension. On peut sinon utiliser un chiffre, cf. ce qu'utilise Carbone 4 cf. http://www.bilans-ges.ademe.fr/forum/viewtopic.php?f=20&t=45&sid=8f0420588409bdd7188baa7aa3f2e043 (http://www.carbone4.com/fr/l_actu_de_carbone_4/co2-voiture-%C3%A9lectrique-contre-voiture-diesel-pas-si-simple)

Voir cette étude ACV de l'ADEME sur la comparaison VT / VE (2013) : https://www.ademe.fr/sites/default/files/assets/documents/90511_acv-comparative-ve-vt-resume.pdf - https://www.ademe.fr/elaboration-selon-principes-acv-bilans-energetiques-emissions-gaz-a-effet-serre-autres-impacts-environnementaux

<img width="642" alt="Capture d’écran 2020-02-11 à 12 24 30" src="https://user-images.githubusercontent.com/91435/74232651-274d5500-4cc1-11ea-8908-0dd71da1abb3.png">

En gros, le VE est à considérer dans un mix-électrique décarboné (la France quoi vs l'Allemagne), et pour plus de 50 000 km a minima mais surtout plus de 80 000 km pour avoir un effet bénéfice sur le climat (p15 de la synthèse) !

<img width="642" alt="Capture d’écran 2020-02-11 à 12 25 48" src="https://user-images.githubusercontent.com/91435/74232722-4ba93180-4cc1-11ea-8160-cd102ceac1cd.png">

S'ajoute un élément clé : l'origine de la fabrication des batteries (sa fabrication est responsable de 35/40% des émissions totales - de la fabrication, puis utilisation, à la fin de vie - du VE) et entre autres le mix-électrique du pays où celles-ci sont fabriquées. 
Autre point intéressant, la projection du scénario 2020. On y est et le poids des véhicules n'a pas baissé par rapport à ce qui est envisagé à la fin.

# Définition de l'hybridation

Il existe plusieurs fonctions d’hybridation :

- le système « Stop & Start® » permet, en ville, d’économiser entre 6 et 15 % de carburant (8 g de CO2/km en moyenne) et de diminuer les nuisances sonores. Ce système peut équiper les voitures à essence comme les Diesel ;
- le freinage récupératif permet une économie de carburant de 15 % ;
- avec l’optimisation du moteur thermique, le moteur électrique évite la surconsommation du moteur thermique au moment de l’accélération ;
- enfin, un véhicule hybride peut être propulsé entièrement en mode électrique sans intervention du moteur thermique (fonction E-drive® sur la Toyota Prius).

Les hybrides « rechargeables » (plug-in hybrid) ont également la possibilité de recharger leurs batteries sur le réseau électrique. Les différents types d’hybrides comportent une ou plusieurs fonctions : Stop & Start® (fonction 1), Stop & Go (fonctions 1 et 2), Mild Hybrid (fonctions 1, 2 et 3) et Full Hybrid (fonctions 1, 2, 3 et 4).

@martinregner tu sais si une autre étude est en cours, ou prévue pour mettre à jour ce comparatif ?

**Comment by laem** [comment]
Je crois que l'étude de l'ADEME de 2013 a été un peu rendue obsolète par celle-ci : http://www.fondation-nature-homme.org/sites/default/files/vehicule_electrique_synthese.pdf


![Annotation 2020-02-11 120256](https://user-images.githubusercontent.com/1177762/74231248-76e24f80-4cc6-11ea-81fd-4ea4a1fa7320.png)

Il y a le même graphique pour une berline.

On y retrouve aussi l'évolution de l'impact en fonction de l'intensité carbone du mix électrique.

Sinon, pour l'international, cet article mis à jour en février : https://www.carbonbrief.org/factcheck-how-electric-vehicles-help-to-tackle-climate-change
On y apprend notamment qu'en UK, si on compte continuer de conduire pendant 4 ans une voiture conventionnelle, il vaut mieux acheter une électrique ! 

![Annotation 2020-02-11 120954](https://user-images.githubusercontent.com/1177762/74231694-739b9380-4cc7-11ea-88d4-3a5060d159fc.png)

**Comment by fjg** [comment]
Oui, mais je ne voulais pas multiplier les sources et je n'ai pas vu de position de l'Ademe vis à vis de cette étude (FNH), même s'il y a son logo et sa participation.
Ensuite ca reparle des mêmes constats (production, mix-électrique, usage... puis recyclage quand on y sera). Le V2G & co c'est par encore d'actualité, même si c'est un élément à tenir compte dans le cadre d'une stratégie. A mon sens à développer dans les "fiches solutions".

**Comment by laem** [comment]
En effet c'est plus une étude Carbone4 que ADEME. En même temps, c'est le cas de bcp d'études  "officielles" ADEME j'ai l'impression.
